tchannel/tornado/connection.py

```diff
@@ -122,6 +141,17 @@ def __init__(self, connection, tchannel=None):
 
         self.tchannel = tchannel
 
+        # Map from message ID to inflight requests on the server side.
+        self.incoming_requests = {}
+        # Map from message ID to inflight responses on the client side.
```

Comment:
An inbound request on the *client* side would still live in the `inflight_requests` map.